src/main/java/hudson/plugins/jira/workflow/CommentStep.java

```diff
@@ -0,0 +1,99 @@
+package hudson.plugins.jira.workflow;
+
+import java.io.IOException;
+
+import javax.annotation.Nonnull;
+
+import org.jenkinsci.plugins.workflow.steps.AbstractStepDescriptorImpl;
+import org.jenkinsci.plugins.workflow.steps.AbstractStepImpl;
+import org.jenkinsci.plugins.workflow.steps.AbstractSynchronousNonBlockingStepExecution;
+import org.jenkinsci.plugins.workflow.steps.StepContextParameter;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import com.google.inject.Inject;
+
+import hudson.Extension;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import hudson.plugins.jira.JiraSession;
+import hudson.plugins.jira.JiraSite;
+import hudson.plugins.jira.Messages;
+
+/**
+ * Simple add comment step.
+ *
+ * @author jan zajic
+ */
+public class CommentStep extends AbstractStepImpl {
+
+    public final String issueKey;
+
+    public final String body;
+
+    @DataBoundConstructor
+    public CommentStep(@Nonnull String issueKey, @Nonnull String body) {
+        this.issueKey = issueKey;
+        this.body = body;
+    }
+
+    public String getIssueKey() {
+        return issueKey;
+    }
+
+    public String getBody() {
+        return body;
+    }
+
+    @Extension
```

Comment:
Should be optional since it comes from the optional plugin
<code>@Extension(optional=true)</code>